.github/workflows/security-center-snippets.yaml

```diff
@@ -19,6 +19,7 @@ on:
     - main
     paths:
     - 'security-center/snippets/v1/**'
+    - 'security-center/snippets/v2/**'
```

Comment:
Please merge #3883 first (path should just be `security-center/snippets/**`).

Reply:
This has now been merged. @vijaykanthm could you update to take this change into account?

Reply:
I have already updated it @grayside.